docs/configuration.md

```diff
@@ -277,17 +277,32 @@ Threshold calculations in order:
 - `mutation score < threshold-low`:
     - Danger! Your reporters will display red colors, you're in the danger zone now.
 - `mutation score < threshold-break`:
-    - Error! The application will exit with exit code 1.
+    - Error! Stryker will exit with exitcode 1.
 
 ### `break-at` <`number`>
 
 Default: `0`  
 Command line: `[-b|--break-at] 40`  
-Config file: `"thresholds": { "break": 0 }`
+Config file: See [thresholds](#thresholds-object)
 
-When threshold break is set to anything other than 0 Stryker will exit with a non-zero code. This can be used in a CI pipeline to fail the pipeline when you mutation score is not sufficient. Must be less than or equal to threshold low.
+Must be less than or equal to threshold low.  
+When threshold break is set to anything other than 0 Stryker will exit with a non-zero code. This can be used in a CI pipeline to fail the pipeline when you mutation score is not sufficient.  
```

Comment:
```suggestion
When threshold break is set to anything other than 0 Stryker will exit with a non-zero code. This can be used in a CI pipeline to fail the pipeline when your mutation score is not sufficient.  
```

Reply:
```suggestion
When threshold break is set to anything other than 0 and the actual mutation score is lower than that, Stryker will exit with a non-zero code. This can be used in a CI pipeline to fail the pipeline when you mutation score is not sufficient.
```